minifi/minifi-c2/minifi-c2-cache/minifi-c2-cache-s3/src/main/java/org/apache/nifi/minifi/c2/cache/s3/S3WritableConfiguration.java

```diff
@@ -81,6 +82,11 @@ public InputStream getInputStream() throws ConfigurationProviderException {
     return s3Object.getObjectContent();
   }
 
+  @Override
+  public URL getURL() throws ConfigurationProviderException {
```

Comment:
I think no ConfigurationProviderException is thrown here.

Reply:
True but it overrides the method which says it could so I just left it in